packages/common/src/encryption.ts

```diff
@@ -1,7 +1,8 @@
 import crypto from "crypto";
 
 const algorithm = "aes-256-cbc"; //Using AES encryption
-const key = Buffer.from("1d71cceced68159ba59a277d056a66173613052cbeeccbfbd15ab1c909455a4d", "hex"); // TODO: generate with const data = crypto.randomBytes(32).toString('hex')
+const encryptonKey = process.env.ENCRYPTION_KEY ?? "0000000000000000000000000000000000000000000000000000000000000000"; // Fallback to a default key for local development
```

Comment:
Can we log a warning in case the fallback value is used? Using fallback values is generally a security risk. We should also warn about this in the UI.

Reply:
Also Typo, `encryptionKey` is missing the "i"

Reply:
The key is generated initally, my plan is to add a cli option that allows you to genrate a new key and also update all existing integration secrets